src/julia/Prechelt/src/Prechelt.jl

```diff
@@ -0,0 +1,76 @@
+#=
+# Port of Peter Norvig's Common Lisp program from http://norvig.com/java-lisp.html.
+#
+# - Julia version: 1.6.2
+# - Author: Renato Athaydes
+# - Date: 2021-07-24
+=#
+module Prechelt
+
+export run_encodings
+
+const emptyStrings = String[]
+
+const KEYS = begin
+    m = [
+         'j' 'r' 'd' 'f' 'a' 'c' 'b' 'l' 'g'
+         'n' 'w' 's' 't' 'm' 'i' 'k' 'o' 'h'
+         'q' 'x' 'y' '_' '_' 'v' 'u' 'p' 'z'
+        ]
+    map('a':'z') do c
+        n = 1
+        for col in eachcol(m) if c in col return n else n += 1 end end
+        0
+    end
+end
+
+function printTranslations(dict, num, digits, start=1, words=String[])
+    if start > length(digits)
+       return println(num, ": ", join(words, " "))
+    end
+    foundWord = false
+    n = BigInt(1)
```

Comment:
A few other people have made optimisations using `Base.GMP` which is basically handing off big-int to C.

https://github.com/renatoathaydes/prechelt-phone-number-encoding/blob/julia-optimisations/src/julia/phone_encoder.jl

Reply:
Someone else even used a pair of u128 I think... in Java/Rust, the fastest implementations use just bytes as keys.